docs/Reference/Console/configuration.md

```diff
@@ -0,0 +1,38 @@
+# Console Configuration
+
+Console can be configured using either a YAML configuration file (`config.yml`) or environment variables (`.env`). Both methods allow you to customize the behavior of the Console application.
+
+## Using config.yml
+
+When you first run the Console BETA, a `config.yml` file is generated and stored in the `config/` directory, located in the same directory as the Console executable. You can stop the Console, edit the config.yml file, and then restart the Console to apply the changes.
```

Comment:
im not sure i see the need to say BETA everwhere, the title is fine, make less work for yourself in the future.